core/src/main/java/apoc/schema/Schemas.java

```diff
@@ -574,14 +574,15 @@ private IndexConstraintRelationshipInfo relationshipInfoFromIndexDescription(Ind
                 .mapToObj(tokens::propertyKeyGetName)
                 .collect(Collectors.toList());
         try {
-            return new IndexConstraintRelationshipInfo(getSchemaInfoName(relName, properties), relName, properties, schemaRead.indexGetState(indexDescriptor).toString());
+            return new IndexConstraintRelationshipInfo(getSchemaInfoName(relName, properties), "INDEX", properties, schemaRead.indexGetState(indexDescriptor).toString(), relName);
         } catch (IndexNotFoundKernelException e) {
             return new IndexConstraintRelationshipInfo(
                     // Pretty print for index name
                     getSchemaInfoName(relName, properties),
-                    relName,
+                    "INDEX",
```

Comment:
above we check:  `!indexDescriptor.isUnique() ? "INDEX" : "UNIQUENESS"`, coming very soon is new relationship constraints that allow this, does this check work here too?

Reply:
Oh you're right, I didn't remember about your card, I was referring to the current version. Updated :)